tools/db_crashtest.py

```diff
@@ -245,7 +250,12 @@ def setup_expected_values_dir():
     if expected_values_dir is not None:
         return expected_values_dir
     expected_dir_prefix = "rocksdb_crashtest_expected_"
-    test_tmpdir = os.environ.get(_TEST_DIR_ENV_VAR)
+    test_tmpdir = os.environ.get(_TEST_EXPECTED_DIR_ENV_VAR)
```

Comment:
This code block is a bit confusing. Can you change test_tmpdir to test_exp_tmpdir or something similar?